main.tf

```diff
@@ -232,273 +236,11 @@ resource "aws_lb_target_group_attachment" "prometheus" {
-    locust_base_url  = "http://${module.internal_nlb.dns_name}:${local.emqx_http_api_port}/api/${local.emqx_api_version}"
-  })
-  filename = "${path.module}/ansible/group_vars/locust.yml"
-}
-
-resource "terraform_data" "ansible_init" {
-  depends_on = [
-    local_file.ansible_inventory
-  ]
-
-  provisioner "local-exec" {
-    command = "ansible-galaxy collection install -r ansible/requirements.yml"
-  }
-  provisioner "local-exec" {
-    command = "ansible-galaxy role install -r ansible/requirements.yml"
-  }
-}
-
-resource "terraform_data" "ansible_playbook_http" {
-  depends_on = [
-    module.integration,
-    terraform_data.ansible_init,
-    local_file.ansible_common_group_vars
-  ]
-  provisioner "local-exec" {
-    command = "ansible-playbook ansible/http.yml"
-    environment = {
-      no_proxy = "*"
-    }
-  }
-}
-
-resource "terraform_data" "ansible_playbook_rabbitmq" {
-  depends_on = [
-    module.integration,
-    terraform_data.ansible_init,
-    local_file.ansible_common_group_vars
-  ]
-  provisioner "local-exec" {
-    command = "ansible-playbook ansible/rabbitmq.yml"
-    environment = {
-      no_proxy = "*"
-    }
-  }
-}
-
-resource "terraform_data" "ansible_playbook_emqx" {
-  depends_on = [
-    terraform_data.ansible_init,
-    local_file.ansible_common_group_vars,
-    local_file.ansible_emqx_group_vars,
-    local_file.ansible_emqx_host_vars
-  ]
-  provisioner "local-exec" {
-    command = "ansible-playbook ansible/emqx.yml"
-    environment = {
-      no_proxy = "*"
-    }
-  }
-}
-
-resource "terraform_data" "ansible_playbook_loadgen" {
-  depends_on = [
-    terraform_data.ansible_init,
-    local_file.ansible_common_group_vars,
-    local_file.ansible_loadgen_group_vars,
-    local_file.ansible_locust_group_vars,
-    local_file.ansible_loadgen_host_vars
-  ]
-  provisioner "local-exec" {
-    command = "ansible-playbook ansible/loadgen.yml"
-    environment = {
-      no_proxy = "*"
-    }
-  }
-}
-
-resource "terraform_data" "ansible_playbook_tuning" {
-  depends_on = [
-    terraform_data.ansible_init,
-    local_file.ansible_common_group_vars
-  ]
-  provisioner "local-exec" {
-    command = "ansible-playbook ansible/tuning.yml"
-  }
-}
-
-resource "terraform_data" "ansible_playbook_monitoring" {
-  depends_on = [
-    terraform_data.ansible_init,
-    local_file.ansible_common_group_vars
-  ]
-  provisioner "local-exec" {
-    command = "ansible-playbook ansible/monitoring.yml"
-    environment = {
-      no_proxy = "*"
-    }
-  }
-}
-
-resource "terraform_data" "ansible_playbook_node_exporter" {
-  depends_on = [
-    terraform_data.ansible_init,
-    local_file.ansible_common_group_vars
-  ]
-  provisioner "local-exec" {
-    command = "ansible-playbook ansible/node_exporter.yml"
-    environment = {
-      no_proxy = "*"
-    }
+module "certs" {
+  source = "./modules/certs"
```

Comment:
This seems to be missing from the repo/branch?

Reply:
oops, fixed